rfcs/additional_cypto_cbor_libs.md

```diff
@@ -0,0 +1,38 @@
+# RFC 208: additional crypto and CBOR libraries
+
+## Summary
+
+Several new web APIs require additional cryptography and CBOR libraries to
+properly test. These libraries are to support cryptography and the CBOR
+protocols not otherwise supported in JavaScript or Python, namely Ed25519,
+HPKE, and CBOR. There are open source libraries commonly available that
+implement these protocols and have compatible licenses. This RFC proposes
+adding such libraries to the tools/ directory so that web-platform-tests may
+exercise and verify proper compatible implementations of these new web APIs.
+
+## Details
+
+We're proposing adding these libraries (or some very similar ones) to the
+tools/ directory:
```

Comment:
JS stuff belongs in `/resources` (if it's expected to be maintained as core WPT infrastructure — which probably isn't true in this case, I'd expect these to be maintained by people working on related features), or in `/common`.

See also #46.

Reply:
So having it in third_party/ under a specific subdir sounds great until it's needed in more places... which may mean it's the right way of doing it until it's actually needed in more places rather than speculated to be...?

Reply:
I updated the RFC to follow the advice here: https://github.com/web-platform-tests/rfcs/issues/46#issuecomment-587707539